spacy/pipeline/tok2vec.py

```diff
@@ -82,6 +82,15 @@ def add_listener(self, listener: "Tok2VecListener", component_name: str) -> None
         self.listener_map.setdefault(component_name, [])
         self.listener_map[component_name].append(listener)
 
+    def remove_listener(self, listener: "Tok2VecListener", component_name: str) -> None:
```

Comment:
⚠️ Important note: We need to remember to add this to the `Transformer` component as well.

Reply:
I'll just go ahead and create that PR now